recipes/plate-simulation/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "plate-simulation" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.10" %}
+{% set python_max = "3.11" %}
```

Comment:
This makes the recipe a bit hard to read and maintain.
The min has a special meaning b/c it can be conda-forge default min, Python EoL, or a higher min is necessary, like in this case.

```suggestion
```